tests/test_storage_bucket_file.py

```diff
@@ -94,8 +95,8 @@ def test_properties(self, exo, bucket, tmp_path):
                 "Grants": [
                     {
                         "Grantee": {
-                            "DisplayName": owner_id,
-                            "ID": owner_id,
+                            "DisplayName": owner,
+                            "ID": owner,
```

Comment:
I am a bit surprised you have same info for ID/DisplayName
DisplayName should be Organization name and ID should be Organization UUID

Reply:
@bcachet you mean that I should have `owner_id = ["Owner]["ID"]` and `owner_displayname = ["Owner]["DisplayName"]`?

Reply:
If you are in such a configuration it is because bucket has been created after we deploy https://github.com/exoscale/pithos-private/pull/416

Reply:
Sorry for the noise, you are doing a `put-object-acl` I thought it was result from `get-object-acl`
So everything is fine here